File: Anul-3/Semestrul-I/PDM/IonicAngular/NodeJS/src/model/movie.ts
export interface Movie{
    id: number,
    name: string,
    premierDate: Date,
    rating: number,
    running: boolean,
    owner_id: number,
    photoPath?: string | null,
    photoUrl?: string | null,
    latitude?: number | null,
    longitude?: number | null,
    locationLabel?: string | null
}

File: Anul-3/Semestrul-I/PDM/Android/Frontend/app/src/main/java/com/example/frontend/Movie.kt
package com.example.frontend

import com.google.gson.annotations.SerializedName
import java.util.Date

data class Movie(
    val id: Int,


    var name: String,

    var premierDate: Date,
    var rating: Double,
    val running : Int,
    val owner_id: Int,

    @SerializedName("photoPath")
    var imagePath: String? = null
)